src/audio/copier/copier.c

```diff
@@ -1297,6 +1297,27 @@ static void update_internal_comp(struct comp_dev *parent, struct comp_dev *child
 	child->direction = parent->direction;
 }
 
+static void update_buffer_format(struct comp_buffer __sparse_cache *buf_c,
```

Comment:
and you have a typo in the commit message. Maybe its better to say "Use helper function to update buffer format and avoid code duplication"

Reply:
thanks